crud.py

```diff
@@ -0,0 +1,70 @@
+from sqlalchemy.orm import Session
+
+import models
+import schemas
+
+
+def get_all_authors(
+        db: Session,
+        skip: int | None,
+        limit: int | None = None,
+) -> list[models.Author]:
+    query = db.query(models.Author)
+
+    if limit:
+        query = query.offset(limit)
+
+    if skip:
+        query = query.offset(skip)
```

Comment:
The same issue appears here. You are using the 'skip' variable to offset your query results. This means that if you had previously offset your results, you will offset them again. This is likely not the intended behavior. Please, check this logic.